src/components/cuda/cupti_profiler.c

```diff
@@ -2613,3 +2869,41 @@ static int evt_name_to_device(const char *name, int *device)
     }
     return PAPI_OK;
 }
+
+/** @class evt_name_to_stat
+  * @brief Take a Cuda native event name with a stat qualifer appended to 
+  *        it and collect the stat .
+  * @param *name
+  *   Cuda native event name with a stat qualifier appended.
+  * @param *stat
+  *   Stat collected.
+*/
+static int evt_name_to_stat(const char *name, int *stat, const char *base)
+{
+    int htable_errno, papi_errno;
+    cuptiu_event_t *event;
+    char *p = strstr(name, ":stat=");
+    if (p) {    
+      p += 6; // Move past ":stat="
+      int i;
+      for (i = 0; i < NUM_STATS_QUALS; i++) {
+          size_t token_len = strlen(stats[i]);
+          if (strncmp(p, stats[i], token_len) == 0) {
+                *stat = i;
+                return PAPI_OK;
+            }
+        }
+    } else {
+        htable_errno = htable_find(cuptiu_table_p->htable, base, (void **) &event);
+        if (htable_errno != HTABLE_SUCCESS) {
+            papi_errno = (htable_errno == HTABLE_ENOVAL) ? PAPI_ENOEVNT : PAPI_ECMP;
+            return PAPI_ENOEVNT;
```

Comment:
You perform the ternary operator above, but then just return `PAPI_ENOEVNT`. Therefore, I think you could just delete the ternary operator.